sms/send-signed-sms.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+use ;
```

Comment:
Is something missing here? Or was it placed here by accident?

Reply:
Accident, removed. This and `send-sms.php` were used for a lot of ancillary testing of other things so some leftover code got committed.